Server/JokesOnYou.Web.Api/Middlewares/ExceptionMiddleware.cs

```diff
@@ -36,6 +36,7 @@ public async Task InvokeAsync(HttpContext context)
                     AppException => (int)HttpStatusCode.BadRequest,
                     UserRegisterException => (int)HttpStatusCode.BadRequest, //consider using built in ArgumentException. 
                     UserLoginException => (int)HttpStatusCode.Unauthorized,
+                    WrongCredentialsException => (int)HttpStatusCode.Forbidden,
```

Comment:
Shouldnt this be a 401 unauthorized?

Reply:
One Line above there is also the user login exception which could be used.